File: pythonlinkedlists/LinkedListTest1.py
#simple program for creation and  traversal of linkedlists
class Node:
    def __init__(self,data):
        self.data=data
        self.next=None
class LinkedList:
    def  __init__(self):
        self.head=None
    def printList(self):
        temp=self.head
        while(temp!=None):
            print(temp.data)
            temp=temp.next
if __name__=='__main__':
    mylist=LinkedList()
    mylist.head=Node(1)
    second=Node(2)
    third=Node(3)
    mylist.head.next=second
    second.next=third
    mylist.printList()
    
        
    
 

 
 

File: JAVA LINKEDLISTS/LinkedListTest1.java
class LinkedListTest1
{
  static class Node
 { int data;
 Node next;
 Node(int d)
 { data=d;
 next=null;
 }
 }

public static void main(String args[])
{ 

Node head=new Node(1);
Node second=new Node(2);
Node third=new Node(3);
head.next=second;
second.next=third;
third.next=null;
Node pointer=head;
while(pointer!=null)
{ System.out.printf("%d ",pointer.data);
pointer=pointer.next;
}
}
}